lms/djangoapps/instructor/views/api.py

```diff
@@ -1493,9 +1493,17 @@ def post(self, request, course_id, csv=False):  # pylint: disable=redefined-oute
             query_features = [
                 'id', 'username', 'name', 'email', 'language', 'location',
                 'year_of_birth', 'gender', 'level_of_education', 'mailing_address',
-                'goals', 'enrollment_mode', 'last_login', 'date_joined', 'external_user_key'
+                'goals', 'enrollment_mode', 'last_login', 'date_joined', 'external_user_key',
+                'enrollment_date',
             ]
 
+        custom_attributes = configuration_helpers.get_value_for_org(
+            course_key.org,
+            "student_profile_download_custom_student_attributes"
+        )
+        if custom_attributes:
+            query_features.extend(custom_attributes)
```

Comment:
The setting is being pulled out of a json field, so the type is not guaranteed to be a list, we should do some type checking or error handling around this to better handle misconfigurations.  I think throwing an error to the user makes the most sense.